package.json

```diff
@@ -66,7 +68,6 @@
     ]
   },
   "browser": {
-    "bs58": false,
```

Comment:
Since using forge in tests, revert this back to false.

Reply:
@davidlehn The multibase/multicodec tools use Buffer, that's why it's left in: https://github.com/multiformats/js-multicodec/blob/master/src/index.js#L28

Reply:
This is in reference to your comment about `Buffer`